nilearn/glm/second_level/second_level.py

```diff
@@ -606,6 +606,7 @@ def non_parametric_inference(
     verbose=0,
     threshold=None,
     tfce=False,
+    flm_object=False
```

Comment:
Although this fixes the test, I am not a big fan of this approach.

Would be better to not add extra arguments to the API.